relay_hub/server.py

```
@@ -0,0 +1,331 @@
+            return {
+                "status": "ok",
+                "time": f"{datetime.now(timezone.utc).isoformat()}Z",
+                "storage": self.config.get("storage", {}).get("path", "relay.db"),
+            }
+
+    # Lifecycle ---------------------------------------------------------
+
+    async def run(self) -> None:
+        server_cfg = self.config.get("server", {})
+        transport = server_cfg.get("transport", "http")
+        host = server_cfg.get("host", "0.0.0.0")
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)